### DISCUSSION TITLE
Flutter (missing UUID 4c4c441555553144a1123c0fff4c00d2)

### DISCUSSION THREAD

**Discussion by KevinSnoopy** [firebase/flutterfire#11143] [Q&A]
Flutter
(missing UUID 4c4c441555553144a1123c0fff4c00d2)

I confirm that all DSYMs have been uploaded successfully, but it still shows that dsym is missing。

firebase_crashlytics: ^3.3.1

**Comment by dinh-nguyen-ominext** [comment]
@KevinSnoopy 
Have you found this missing UUID yet ?
If you find it, please let me know .
